### PR TITLE
Aug 1 fixes

### PR DESCRIPTION
**Pull request by nvmbrasserie**

### CODE REVIEW

File: datasets/fr/tresor/fr_tresor_gels_avoir.yml [hunk 1]
```diff
@@ -4353,3 +4353,29 @@ lookups:
             - 1987
             - 2001
           idNumber: 10102191894
+      - match: "Date de création: 15.12.2023 - Lieu de création: Kenya: /"
```
Comment (jbothma) [inline datasets/fr/tresor/fr_tresor_gels_avoir.yml:4356]: I think we already have all of these, don't we?

Comment (nvmbrasserie) [reply datasets/fr/tresor/fr_tresor_gels_avoir.yml:4356]: there was a bunch of new ones today morning, I think they're different

Comment (jbothma) [reply datasets/fr/tresor/fr_tresor_gels_avoir.yml:4356]: this branch turns 2 into 3. Not saying I haven't caused some duplicates :)

**Comment by jbothma** [comment]
I blatantly plagiarised this https://github.com/opensanctions/opensanctions/pull/1114

**Pull request closed without merging**